src/streamable/bytes.rs

```diff
@@ -115,14 +117,22 @@ impl<'a, const N: usize> From<&'a BytesImpl<N>> for &'a [u8] {
 }
 
 impl<const N: usize> BytesImpl<N> {
-    pub fn slice(&self) -> &[u8; N] {
-        &self.0
-    }
     pub fn to_vec(&self) -> Vec<u8> {
         self.0.to_vec()
     }
 }
 
+impl<const N: usize> AsRef<[u8]> for BytesImpl<N> {
+    fn as_ref(&self) -> &[u8] {
+        &self.0
+    }
+}
+impl<const N: usize> Deref for BytesImpl<N> {
+    type Target = [u8];
+    fn deref<'a>(&'a self) -> &'a [u8] {
+        &self.0
```

Comment:
Can you fix this clippy warning?

Reply:
I thought clippy warnings would fail CI. I think github might be using a newer version than we use in CI.